### PR TITLE
Add jquery elevatezoom plugin

### PR DESCRIPTION
**Pull request by ethanpil**

### CODE REVIEW

File: files/elevatezoom/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "elevatezoom",
+  "repo": "elevateweb/elevatezoom",
+  "files": {
+    "include": ["jquery.elevatezoom.js"]
```
Comment (megawac) [inline files/elevatezoom/update.json]: This file isn't minified and the files they provide are don't follow naming conventions. Mind removing the update.json and instead including the `jquery.elevateZoom-3.0.8.min.js` file (under `3.0.8/jqueryElevateZoom.js` or similar)

**Comment by ethanpil** [comment]
I made the change.

File: files/elevatezoom/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "Andrew Eades"
+github = "https://github.com/elevateweb/elevatezoom"
+homepage = "http://www.elevateweb.co.uk/image-zoom"
+description = "A jQuery image zoom plugin with many options and features."
+mainfile = "3.0.8/jqueryElevateZoom.js"
```
Comment (megawac) [inline files/elevatezoom/info.ini]: remove the `3.0.8`

**Comment by megawac** [comment]
@jsdelivrbot squishity squash

**Pull request closed without merging**